From c00860288817a56130b93c07f8c6c7c6d376a37a Mon Sep 17 00:00:00 2001
From: Sean McGinnis <sean.mcginnis@gmail.com>
Date: Sat, 18 Apr 2020 11:55:23 -0500
Subject: [PATCH 1/3] Use unittest.mock instead of third party mock

Now that we no longer support py27, we can use the standard library
unittest.mock module instead of the third party mock lib.

Change-Id: Iee2ca363966243826134c1b5f5c9762d4b53662f
Signed-off-by: Sean McGinnis <sean.mcginnis@gmail.com>
(cherry picked from commit 9e3c61b6846f842b8b6b3788c002de7dc0d22457)
---
 muranopkgcheck/tests/test_error.py              | 2 +-
 muranopkgcheck/tests/test_log.py                | 2 +-
 muranopkgcheck/tests/test_manager.py            | 2 +-
 muranopkgcheck/tests/test_manifest_validator.py | 2 +-
 muranopkgcheck/tests/test_muranopl_validator.py | 2 +-
 muranopkgcheck/tests/test_package.py            | 2 +-
 muranopkgcheck/tests/test_pkg_loader.py         | 3 ++-
 muranopkgcheck/tests/test_plugin.py             | 2 +-
 muranopkgcheck/tests/test_ui_validator.py       | 2 +-
 muranopkgcheck/tests/test_yaml_validator.py     | 2 +-
 10 files changed, 11 insertions(+), 10 deletions(-)

diff --git a/muranopkgcheck/tests/test_error.py b/muranopkgcheck/tests/test_error.py
index 6ed52a1..fd9f795 100644
--- a/muranopkgcheck/tests/test_error.py
+++ b/muranopkgcheck/tests/test_error.py
@@ -12,7 +12,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
+from unittest import mock
 
 from muranopkgcheck import error
 from muranopkgcheck.tests import base
diff --git a/muranopkgcheck/tests/test_log.py b/muranopkgcheck/tests/test_log.py
index 172bde1..d3fc94a 100644
--- a/muranopkgcheck/tests/test_log.py
+++ b/muranopkgcheck/tests/test_log.py
@@ -12,7 +12,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
+from unittest import mock
 
 from muranopkgcheck import log
 from muranopkgcheck.tests import base
diff --git a/muranopkgcheck/tests/test_manager.py b/muranopkgcheck/tests/test_manager.py
index 2741dd9..6c74dc3 100644
--- a/muranopkgcheck/tests/test_manager.py
+++ b/muranopkgcheck/tests/test_manager.py
@@ -12,7 +12,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
+from unittest import mock
 
 from muranopkgcheck import error
 from muranopkgcheck import manager
diff --git a/muranopkgcheck/tests/test_manifest_validator.py b/muranopkgcheck/tests/test_manifest_validator.py
index 1dcb5ae..f018924 100644
--- a/muranopkgcheck/tests/test_manifest_validator.py
+++ b/muranopkgcheck/tests/test_manifest_validator.py
@@ -13,7 +13,7 @@
 #    under the License.
 
 
-import mock
+from unittest import mock
 
 from muranopkgcheck.tests import test_validator_helpers as helpers
 from muranopkgcheck.validators import manifest
diff --git a/muranopkgcheck/tests/test_muranopl_validator.py b/muranopkgcheck/tests/test_muranopl_validator.py
index b2d51fb..a814e26 100644
--- a/muranopkgcheck/tests/test_muranopl_validator.py
+++ b/muranopkgcheck/tests/test_muranopl_validator.py
@@ -13,7 +13,7 @@
 #    under the License.
 
 from copy import deepcopy
-import mock
+from unittest import mock
 
 from muranopkgcheck.tests import test_validator_helpers as helpers
 from muranopkgcheck.validators import muranopl
diff --git a/muranopkgcheck/tests/test_package.py b/muranopkgcheck/tests/test_package.py
index e1887e6..9b2ff66 100644
--- a/muranopkgcheck/tests/test_package.py
+++ b/muranopkgcheck/tests/test_package.py
@@ -13,7 +13,7 @@
 #    under the License.
 
 
-import mock
+from unittest import mock
 
 from muranopkgcheck.tests import test_validator_helpers as helpers
 from muranopkgcheck.validators import package
diff --git a/muranopkgcheck/tests/test_pkg_loader.py b/muranopkgcheck/tests/test_pkg_loader.py
index b9f1983..386ff3d 100644
--- a/muranopkgcheck/tests/test_pkg_loader.py
+++ b/muranopkgcheck/tests/test_pkg_loader.py
@@ -12,7 +12,8 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
+from io import BytesIO
+from unittest import mock
 import zipfile
 
 import six
diff --git a/muranopkgcheck/tests/test_plugin.py b/muranopkgcheck/tests/test_plugin.py
index 8ef8806..d88008e 100644
--- a/muranopkgcheck/tests/test_plugin.py
+++ b/muranopkgcheck/tests/test_plugin.py
@@ -12,7 +12,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
+from unittest import mock
 
 from muranopkgcheck import manager
 from muranopkgcheck import plugin
diff --git a/muranopkgcheck/tests/test_ui_validator.py b/muranopkgcheck/tests/test_ui_validator.py
index 280586e..13b12b2 100644
--- a/muranopkgcheck/tests/test_ui_validator.py
+++ b/muranopkgcheck/tests/test_ui_validator.py
@@ -12,7 +12,7 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
+from unittest import mock
 
 from muranopkgcheck.tests import test_validator_helpers as helpers
 from muranopkgcheck.validators import ui
diff --git a/muranopkgcheck/tests/test_yaml_validator.py b/muranopkgcheck/tests/test_yaml_validator.py
index c49227a..2de297f 100644
--- a/muranopkgcheck/tests/test_yaml_validator.py
+++ b/muranopkgcheck/tests/test_yaml_validator.py
@@ -12,8 +12,8 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-import mock
 import unittest
+from unittest import mock
 
 from muranopkgcheck.validators import base
 
-- 
2.35.3

